IM_calculation/Advanced_IM/runlibs_2d.py

```diff
@@ -89,15 +90,50 @@ def datetime_to_file(t_value, t_type: str, out_dir):
 
 # def save_timeout_value(component_outdir, timeout_threshold, datetime.datetime.now()):
 
+def run_and_check(run_script, comp_paths, output_dir, opensees_path, timeout):
```

Comment:
Have the existing models been tested with this change? I wouldn't expect any issues, but a concrete check would be good